tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -76,6 +76,14 @@ spec:
                 containerImage="${deliveryRepo}@${digest}"
                 # purl should be pkg:oci/bar@sha256%3Aabcde?arch=amd64&repository_url=registry.redhat.io/foo
                 purl="pkg:oci/${deliveryRepo##*/}@${digest/:/%3A}?arch=${arch}&repository_url=${deliveryRepo%/*}"
+                uniqueTagRegex="(rhel-)?v?[0-9]+\.[0-9]+(\.[0-9]+)?-[0-9]{8,}"
```

Comment:
Maybe this is good enough, but how about selecting the longest out of such tags? I think that's also how RHEC displays images - if you have 1.1.0-123456 and 1.1-123456, both will be unique, but the longer is more specific. Just a thought. Maybe it doesn't really matter which one is picked.